sdk/core/azure-core/tests/async_tests/test_rest_stream_responses_async.py

```diff
@@ -66,25 +66,25 @@ async def test_iter_bytes(client):
         assert response.is_closed
         assert raw == b"Hello, world!"
 
-@pytest.mark.asyncio
-async def test_iter_text(client):
-    request = HttpRequest("GET", "/basic/string")
+# @pytest.mark.asyncio
+# async def test_iter_text(client):
+#     request = HttpRequest("GET", "/basic/string")
```

Comment:
Why we comment these out rather than removing them? 

Does that mean we plan to add them back in the future?

Reply:
Yeah, planning to add them back in the future. I can delete if people want to remove the tests

Reply:
i was just feeling attached so didn't want to delete

Reply:
Then maybe we can just skip the tests w/ some reasons.

It can help us in the future to understand what happened.